prow/plugins/jira-linker/jira-linker.go

```diff
@@ -18,8 +18,9 @@ const (
 )
 
 var (
-	jiraRegex     = regexp.MustCompile("([A-Z]+)-\\d+")
-	enabledEvents = []github.PullRequestEventAction{
+	jiraTitleRegex  = regexp.MustCompile("([A-Z]+)-\\d+")
+	jiraBranchRegex = regexp.MustCompile("^\\w+/(([A-Za-z]+)-\\d+)(-|$|_)")
```

Comment:
Optional nit: `(?: ...)` for a group you don't need captured (i.e. can do `(?:-|$|_)` at end).

Reply:
Sorry - merged just as you posted this! Thanks for the suggestion - I was trying to use a positive lookahead but they don't exist sadly; I didn't realise this exists 😄

Reply:
(It would still be part of `[0]` in the match result, but wouldn't create a separate entry.)

Reply:
Yeah, it's always frustrated me that I can't do alternatives easily without capturing it -- thanks!